htcanalyze/main.py

```diff
@@ -646,21 +657,26 @@ def print_results(htcanalyze: HTCAnalyze,
 
 def run(commandline_args):
     """
-    Run this script.
+    Run HTCAnalyze
```

Comment:
The first two lines of the comment can be removed. There's nothing else that could be run.